nuntium/models.py

```diff
@@ -55,6 +55,13 @@ class WriteItInstance(models.Model):
         through='Membership')
     owner = models.ForeignKey(User, related_name="writeitinstances")
 
+    def add_person(self, person):
+        Membership.objects.get_or_create(writeitinstance=self, person=person)
+
+    @property
+    def persons_with_contacts(self):
+        return self.persons.filter(contact__isnull=False)
```

Comment:
What happens if a Person has multiple Contacts? I think you're going to get a row per Contact/Person combination. Is that what you want?